app/services/base_entity.rb

```diff
@@ -0,0 +1,11 @@
+class BaseEntity
```

Comment:
This can be removed, right?

Reply:
Actually no. It was removed in the next PR.